CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - The macOS installer no longer overwrites the service files when reinstalling.
 - Cache smart contract modules on startup from existing state to improve smart
   contract execution.
+- Make consensus queries more robust, by validating input more extensively.
```

Comment:
Maybe add the GRPC code that invalid ones will return i.e., `InvalidArgument`?

Reply:
Done.